lib/paperclip/storage/s3.rb

```diff
@@ -232,6 +241,8 @@ def s3_interface
             config[:proxy_uri] = URI::HTTP.build(proxy_opts)
           end
 
+          config[:use_accelerate_endpoint] = true if use_accelerate_endpoint?
```

Comment:
`config[:use_accelerate_endpoint] = true if use_accelerate_endpoint?`

is the same as

`config[:use_accelerate_endpoint] = use_accelerate_endpoint?`